generators/app/templates/src/config/api.ts

```diff
@@ -13,7 +13,7 @@ const api = create({
   timeout: 5000
 });
 
-api.addMonitor(Reactotron.apisauce);
+api.addMonitor(((Reactotron as unknown) as { apisauce: any }).apisauce);
```

Comment:
:magic: :que-trucazo: hahaha

Reply:
I was having a war with this, it was impossible to type this on a nice way, if you know I will be very glad for some help haha